src/Microsoft.AspNet.Mvc.Core/BindAttribute.cs

```diff
@@ -136,5 +140,17 @@ private ObjectFactory GetFactory()
                 return predicate(context, propertyName);
             };
         }
+
+        private static string[] SplitString(string original)
+        {
+            if (string.IsNullOrEmpty(original))
+            {
+                return new string[0];
+            }
+
+            var split = original.Split(',').Select(piece => piece.Trim()).Where(piece => !string.IsNullOrEmpty(piece));
```

Comment:
Look at the overload of `Split` with `StringSplitOptions` - then you don't need any of this ceremony